src/main/java/edu/kis/powp/jobs2d/command/historyComponent/HistoryManager.java

```diff
@@ -0,0 +1,30 @@
+package edu.kis.powp.jobs2d.command.historyComponent;
+
+import edu.kis.powp.jobs2d.command.manager.DriverCommandManager;
+
+import javax.swing.*;
+import java.time.format.DateTimeFormatter;
+
+public class HistoryManager {
+
+    private static final String HISTORY_TIME_PATTERN = "HH:mm:ss";
+    public static DefaultListModel<CommandHistory> historyEntryDefaultListModel;
+    public static DriverCommandManager commandManager;
+
+    public HistoryManager(DriverCommandManager commandManager) {
+        historyEntryDefaultListModel = new DefaultListModel<>();
+        HistoryManager.commandManager = commandManager;
+    }
+
+    public void updateCommandListField() {
```

Comment:
if "commandManager.getCurrentCommandString(), commandManager.getCurrentCommand()" were passed in params. The history could be also independent of commandManager. Extracting dependency to the caller does not introduce any new dependencies.